### ISSUE TITLE
Explicit Mods

### ISSUE THREAD

**Issue by flyweight** [trackpete/exiletools-indexer#151]
Would it be possible to get mods.explicit properties without the Item Type included?  I understand why you did it this way, to help with narrowing your searches and only certain item types have mods, but doing it this way makes my queries incredibly complex.

**Comment by trackpete** [comment]
Sorry for the slow response! So, it's definitely *possible* it's just not something I want to do without a very strong use case for it because it will significantly increase the complexity of the mapping and index.

My theory is that very few people want to search for any item that matches a stat, and if they do, this will most likely be a numeric mod that will match in modsTotal (that doesn't include booleans and certain item specific mods like flasks, maps, etc.).  Generic type-less searching is something I'd actively discourage, because every type can have hugely different mods, min/max values, etc.

Basically I think it makes more sense to force people to choose a list of item types they want to search, rather than allowing it to be open....  but if you really want the latter, you can get most of the functionality using modsTotal.

If you have specific use cases that you're struggling with (other than a generic "I want to search every item type for something with `does something cool on low life` kind of thing"), let me know or post in /r/pathofexiledev I'll see if I can help.
